src/styles/blocks/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
https://github.com/mate-academy/layout_catalog/pull/4078#discussion_r1481114566 - in this comment, it was meant to bring the border box into the general stylization.
Fix this isssue in all styles.